libs/spark/stories/navbar.stories.tsx

```diff
@@ -24,37 +25,32 @@ const UserMenu = styled.div`
 
 const Template = (args: NavBarProps) => (
   <NavBar {...args}>
-    <Toolbar>
-      <PrendaMonogramDark fontSize="62px" />
-      <NavbarLink href="/">
-        <HomeIconDuotone fillColor="transparent" />
-        <span>Dashboard</span>
-      </NavbarLink>
-      <NavbarLink href="/" aria-current="page">
-        <CheckCircleIconDuotone />
-        <span>Active</span>
-      </NavbarLink>
-      <NavbarLink className="disabled">
-        <RocketIconDuotone />
-        <span>Disabled</span>
-      </NavbarLink>
-      <NavbarLink style={{ marginLeft: 'auto' }}>
-        <InboxFilledIconDuotone />
-        <span>0</span>
-      </NavbarLink>
+    <Toolbar style={{ gap: 8 }}>
+      <PrendaMonogramDark fontSize="4.5rem" />
+      <NavBarButton href="" startIcon={<HomeIconDuotone />} aria-current="page">
```

Comment:
```suggestion
      <NavBarButton href="#" startIcon={<HomeIconDuotone />} aria-current="page">
```
`href`'s shouldn't be blank. To not go anywhere, just supply a hash.